test/action_test.dart

```diff
@@ -118,21 +117,20 @@ void main() {
         var action = Action();
         var asyncListenerCompleted = false;
         action.listen((_) async {
-          await Future.delayed(Duration(milliseconds: 100), () {
+          await Future.delayed(const Duration(milliseconds: 100), () {
```

Comment:
I thought the `const` would be optional at this point, is it not?

Reply:
It was suggested by the dart linter: 
```
Use 'const' with the constructor to improve performance.
Try adding the 'const' keyword to the constructor invocation.
dartprefer_const_constructors
```